doc/installation.rst

```diff
@@ -13,7 +13,7 @@ VM image
 The best and easiest way to install Camisole is to download a Virtualbox
 export here:
 
-`Download camisole-latest.ova
+`Download camisole-latest.ova (1.18 GB)
```

Comment:
https://www.engadget.com/2010/12/16/32-gb-versus-32gb-almost-everyone-is-writing-it-wrong/